src/themes/elife/styles.css

```diff
@@ -112,3 +111,10 @@ main {
   text-align: center;
   text-transform: capitalize;
 }
+
+@media only screen and (min-width: 45.625em) {
```

Comment:
We should take advantage of the [`postcss-custom-media`](https://github.com/postcss/postcss-custom-media) plugin [used by Thema](https://github.com/stencila/thema/blob/f966b5b0f6152e88d179c7639db9aa4f627fc464/postcss.config.js#L12) to eliminate the magic numbers of the breakpoint. Can then use a name derived from the eLife breakpoint names, e.g. this one could be `medium`, derived from https://github.com/elifesciences/pattern-library/blob/1a8e5c72d6c22fbb1d9195b395c55bb7e5dcde78/assets/sass/_variables.scss#L118. Seems okay to set the value in `em` as it has been here.

Reply:
I have changed this to wide as this fits what was 45.625em in terms of how the page changed.